src/clean_air/util/file_converter.py

```diff
@@ -38,11 +41,73 @@ def generate_dataframe(filepath):
         # that into a pandas dataframe.
         temp_dataset = xr.open_dataset(filepath)
         temp_dataframe = temp_dataset.to_dataframe()
+    elif filepath.endswith(".csv"):
```

Comment:
That sounds quite useful.  I might make it into a ticket so that we can replace all of these use cases with MIME determinations; it just seems like a less error-prone approach.

https://metoffice.atlassian.net/jira/software/projects/CAP/boards/213/backlog?issueParent=186867&selectedIssue=CAP-326